config/routes.rb

```diff
@@ -11,6 +11,7 @@
   root 'calculators#index'
   get '/calculator', to: "calculators#calculator"
   get '/about_us', :to => redirect('/about_us.html')
+  get '/contact_us', to: "contact_us#index" 
```

Comment:
No need for separate endpoint, if you are using default action names
```suggestion
  resources :contact_us, only: [:index]
```